.travis.yml

```diff
@@ -32,7 +32,11 @@ script: |
   else
     yarn ci
   fi
-after_failure: |
-  if [ "${TRAVIS_EVENT_TYPE}" == "cron" ]; then
-    curl -X POST --data-urlencode "payload={\"attachments\": [{\"color\": \"danger\", \"fallback\": \"Build Failure: $TRAVIS_JOB_WEB_URL\", \"title\": \"Build Failure\", \"text\": \"$TRAVIS_JOB_WEB_URL\"}]}" $SLACK_URL
-  fi
+notifications:
+  slack:
+    if: branch = master AND type = cron
```

Comment:
We didn't have this `master` check before :thinking:

Reply:
We actually don't need this check, as only primary check needed is `type=cron`, have updated the condition